smart-contracts/scripts/deploy-governor.js

```diff
@@ -0,0 +1,86 @@
+const { ethers, upgrades } = require('hardhat')
+const OZ_SDK_EXPORT = require('../openzeppelin-cli-export.json')
+
+const { getNetworkName } = require('../helpers/network')
+const { getDeployment, addDeployment } = require('../helpers/deployments')
+
+async function main() {
+  const [unlockOwner, proposer, executor] = await ethers.getSigners()
+
+  // fetch chain info
+  const chainId = await unlockOwner.getChainId()
+  const networkName = getNetworkName(chainId)
+
+  // eslint-disable-next-line no-console
+  console.log(
+    `Deploying Timelock on ${networkName} with the account: ${unlockOwner.address}...`
+  )
+
+  // deploying Unlock Protocol with a proxy
+  const UnlockDiscountTokenTimelock = await ethers.getContractFactory(
+    'UnlockDiscountTokenTimelock'
+  )
+
+  // one week in seconds
+  const MINDELAY = 60 * 24 * 7
+
+  const timelock = await upgrades.deployProxy(UnlockDiscountTokenTimelock, [
+    MINDELAY,
+    [proposer.address],
+    [executor.address],
```

Comment:
need to define which adresses go there

Reply:
ok this is described here https://docs.openzeppelin.com/contracts/4.x/governance#timelock